llvm/include/llvm/Target/GlobalISel/Combine.td

```diff
@@ -412,7 +412,7 @@ def binop_right_undef_to_undef: GICombineRule<
 
 def unary_undef_to_zero: GICombineRule<
   (defs root:$root),
-  (match (wip_match_opcode G_ABS):$root,
+  (match (wip_match_opcode G_ABS, G_ZEXT):$root,
```

Comment:
One version of this PR combined into G_ZEXT. Here, we are combining G_ZEXT of undef into 0. It relies on the helper
```
 Helper.replaceInstWithConstant(*${root}, 0); 
```